design/gateway-apis-implementation.md

```diff
@@ -51,7 +51,6 @@ There should never be annotations required for functionality on Gateway API obje
 ## Non Goals
 - No TCPRoute or UDPRoute support, that is, no support for arbitrary TCP/UDP forwarding.
```

Comment:
Is there a plan on supporting this in the future?

Reply:
@xaleeks @youngnick ^^